scripts/fbi/hate_crime/utils.py

```diff
@@ -108,3 +111,124 @@ def update_statvar_dcids(statvar_list: list, config: dict):
         ignore_props = get_dpv(d, config)
         dcid = get_statvar_dcid(d, ignore_props=ignore_props)
         d['Node'] = dcid
+
+
+def agg_hate_crime_df(df: pd.DataFrame,
```

Comment:
psl add some unit tests for these new functions with a simple df and a basic flatten, aggregation.

Reply:
added a unit test.